Makefile

```diff
@@ -7,7 +7,8 @@ export DOCKER_CLI_EXPERIMENTAL=enabled
 ## Required to have docker build output always printed on stdout
 export BUILDKIT_PROGRESS=plain
 
-export ARCH=$(arch)
+current_arch := $(shell uname -m)
+export ARCH ?= $(shell case $(current_arch) in (x86_64) echo "amd64" ;; (i386) echo "386";; (aarch64) echo "arm64" ;; (armv6*) echo "arm/v6";; (armv7*) echo "arm/v7";; (ppc64*|s390*|riscv*) echo $(current_arch);; (*) echo "UNKNOWN-CPU";; esac)
```

Comment:
`uname -m` returns `arm64` on mac m1, should we just have the fallback case passthrough the current arch?

Reply:
Let me fix that on the next commit

Reply:
should be done with the last commit. I've laucnhed the full "`make build test`" locally

Reply:
not pushed?

Reply:
now it is

Reply:
output is:

```
+ docker buildx bake -f docker-bake.hcl --load --set '*.platform=linux/arm64' linux
```

which ends up building the alpine image which is not arm64 compatible

Reply:
I got this:

```shell
 uname -m
arm64
➜  docker git:(parallel-multi-arch-tweaks) make build-alpine_jdk8
+ make --silent show
+ jq -r '.target | path(.. | select(.platforms[] | contains("linux/arm64"))?) | add'
Error: the image 'alpine_jdk8' does not exist in manifest for the platform 'linux/arm64'. Please check the output of 'make list'. Exiting.
make: *** [build-alpine_jdk8] Error 1
```

=> it means that something is missing for the `make build`. Let me check this after the lunch

Reply:
OH I see: the flag `-set` overrides the content from the bake file: it is not a restriction such as `docker buildx build --platform <...> ...`. Damn

Reply:
Short term: iterate over the `make --silent list` result.

Reply:
Gotcha, found a solution: instead of calling the group `linux`, let's call the list of images from `make --silent list` which is the filtered list for the current arch.

Reply:
let me know what you think :)